lib/lightning_web/live/profile_live/common_components.ex

```diff
@@ -0,0 +1,90 @@
+defmodule LightningWeb.ProfileLive.CommonComponents do
```

Comment:
@jyeshe `CommonComponents` is really a great name and I like it. But we usually name these modules as `Common` or `Components`, I am wondering if we shouldn't keep that naming for consistency ?

Reply:
done